### DISCUSSION TITLE
Voice acting localization.

### DISCUSSION THREAD

**Discussion by johndoe71rus** [diasurgical/DevilutionX#1262] [General]
If I understand correctly, the game loads the resource files in alphabetical order. As an example, hellfire files.
If I have a version of the game with localized audio files, can I extract and pack them into a new archive diabvoice.mpq?
Will the game first read resources from diabdat.mpq and then connect resources from diabvoice.mpq?

**Comment by AJenbo** [comment]
It does not load arbitrary mpq files. This is the order it looks for content:

- Files on disk (ie, you do not need to put things in an mpq file)
- devilutionx.mpq
- hfopt2.mpq
- hfopt1.mpq,
- hfvoice.mpq,
- hfmusic.mpq
- hfbarb.mpq
- hfbard.mpq
- hfmonk.mpq
- hellfire.mpq
- patch_rt.mpq/patch_sh.mpq
- diabdat.mpq/spawn.mpq
